### ISSUE TITLE
efficientnet_b0 acc1 = 0.65032

### ISSUE THREAD

**Issue by vicwer** [rwightman/gen-efficientnet-pytorch#37]
I load the efficientnet_b0 by `model = torch.hub.load('rwightman/gen-efficientnet-pytorch', 'efficientnet_b0', pretrained=True)`
then I convert the model to onnx. and the eval result is top1_acc=0.65032, top5_acc=0.8718

**Comment by rwightman** [comment]
not enough detail for anyone to help you, most likely an issue with image preprocessing or possibly a version specific bug, there have been varying degrees of issues over various PyTorch / ONNX version combos, but I have successfully converted an onnx model with no accuracy loss
